docs/topics/mta-web-config-git-rep.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="mta-web-config-git-rep_{context}"]
+= Configuring repositories
+
+This management module configures repositories used by {ProductName} ({ProductShortName}) {WebName} with the following options:
+
+* Configuring Git repositories
+* Configuring Subversion repositories
+* Configuring Maven repositories
+
+== Configuring Git repositories
```

Comment:
Only git repositories is detailed here. What about Subversion/Maven?

Reply:
Subversion/Maven were assigned different Jiras and they are in different modules.